demo-shell/src/app/components/cloud/people-groups-cloud-demo.component.html

```diff
@@ -40,16 +40,29 @@
                 [appName]="peopleAppName"
                 [roles]="peopleRoles"
                 [title]="'ADF_TASK_LIST.START_TASK.FORM.LABEL.ASSIGNEE'"
-                [mode]="peopleMode"></adf-cloud-people>
+                [mode]="peopleMode"
+                (selectUser)="onSelectUser($event)"
+                (warning)="onUsersWarning($event)"></adf-cloud-people>
         </div>
 
         <div class="app-people-list" *ngIf="canShowPeopleList()">
+            <h4>All Selected Users: </h4>
             <mat-list role="list">
                 <mat-list-item *ngFor="let item of preSelectUsers" role="listitem">
                     <mat-icon mat-list-icon>person</mat-icon>
-                    {{ item?.firstName }} {{ item?.lastName }}
+                   {{ item?.firstName || item?.lastName || item.username }}
```

Comment:
seems too much in the html template, please move to the code, for example `{{ fromatName(item) }}`

Reply:
Used adf-fullname pipe instead of the method. Thanks